cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -168,6 +170,31 @@ public void query(HttpRequest request, HttpResponder responder,
     }
   }
 
+  @POST
+  @Path("/delete")
+  public void delete(HttpRequest request, HttpResponder responder,
+                    @QueryParam("context") String context,
+                    @QueryParam("metric") String metric,
+                    @QueryParam("start") @DefaultValue("0") final long startTs,
+                    @QueryParam("end") @DefaultValue("2147483647") final long endTs) throws Exception {
```

Comment:
Why is default endtime arbitrarily ending on 19th Jan 2038? Can it be Long.MAX_VALUE ?